### DISCUSSION TITLE
Is the root div #__next gone in next13 when using app folder?

### DISCUSSION THREAD

**Discussion by guitheengineer** [vercel/next.js#45541] [Help] [answered]
I didn't find any documentation on this, but it seems that when using the app folder it doesn't render the root element anymore:

![image](https://user-images.githubusercontent.com/49620737/216495668-35874a5f-1bea-47a0-9135-92a63322cfac.png)

Is this expected?

**Comment by icyJoseph** [comment, marked as answer]
Japp it's gone :(

At least there's no more random `divs` breaking layouts and such. 

On a pet project where I had, sadly, written a couple of CSS rules that depended on the `__next` selector, I figured, since I have control over the entire document layout, through the root `layout.tsx`, I might as well just, for now, wrap my app on a `<div id="__next"></div>`:

```tsx
import "nes.css/css/nes.min.css";
import "design-system/global.css";

import type { ReactNode } from "react";

import { Navigation } from "components/Navigation";
import app from "design-system/App.module.css";

const RootLayout = ({ children }: { children: ReactNode }) => {
  return (
    <html lang="en">
      <head>
        <meta charSet="UTF-8" />
        <meta
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
          name="viewport"
        />
      </head>

      <body>
        <div id="__next">
          <header className={app.header}>
            <Navigation />
          </header>

          <main className={app.content}>{children}</main>
        </div>
      </body>
    </html>
  );
};

export default RootLayout;
```

Sort-of like this. I have to admit that I like the fine-grained control though, but it might be easy to forget certain tags and such. I haven't checked for a while now, but I am not sure if the framework injects some meta-tags it used to, by default.

**Comment by Krak86** [comment]
I see in next 14 it's still gone. For my needed I manually added `id="__next"`
